### PR TITLE
Resolve XML import error with decimal items

### PR DESCRIPTION
**Pull request by Dobbie93**
Ensure the XML import parses decimal entries correctly and resolves "Cannot be greater than 90 degrees" error when importing decimal latitudes

### CODE REVIEW

**Comment by Dobbie93** [comment]
When waypoint is set to Zero Elevation for AH64, it will use the AH64 inbuilt elevation

File: dcs-dtc/Models/AH64/AH64Configuration.cs [hunk 1]
```diff
@@ -99,10 +99,10 @@ internal static AH64Configuration FromCombatFliteXML(AH64Configuration previous,
 
                 var lat = pos.Element("Latitude")?.Value;
                 var lon = pos.Element("Longitude")?.Value;
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
+                var dLat = double.Parse(lat.Replace(',', '.'));
```
Comment (PieterL1976) [inline dcs-dtc/Models/AH64/AH64Configuration.cs:102]: This change doesn't work on systems that use decimal point.
As the is no sure way to check which format is used in the combatflite XML i know of, can be checked as follows:

                double dLat, dLon;
                float elevation;
                if (double.Parse(lon.Replace(',', '.')) > 361)
                {
                    dLat = double.Parse(lat.Replace('.', ','));
                    dLon = double.Parse(lon.Replace('.', ','));
                    float.TryParse(pos.Element("Altitude")?.Value.Replace('.', ','), out elevation);
                }
                else
                {
                    dLat = double.Parse(lat.Replace(',', '.'));
                    dLon = double.Parse(lon.Replace(',', '.'));
                    float.TryParse(pos.Element("Altitude")?.Value.Replace(',', '.'), out elevation);
                }

File: dcs-dtc/Models/F16/F16Configuration.cs [hunk 2]
```diff
@@ -140,10 +140,10 @@ internal static F16Configuration FromCombatFliteXML(F16Configuration previous, s
 
                 var lat = pos.Element("Latitude")?.Value;
                 var lon = pos.Element("Longitude")?.Value;
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
+                var dLat = double.Parse(lat.Replace(',', '.'));
```
Comment (PieterL1976) [inline dcs-dtc/Models/F16/F16Configuration.cs:143]: same as AH64D

File: dcs-dtc/Models/FA18/FA18Configuration.cs [hunk 3]
```diff
@@ -129,9 +129,9 @@ internal static FA18Configuration FromCombatFliteXML(FA18Configuration previous,
 
                 var lat = pos.Element("Latitude")?.Value;
                 var lon = pos.Element("Longitude")?.Value;
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
-                float.TryParse(pos.Element("Altitude")?.Value.Replace('.', ','), out var elevation);
+                var dLat = double.Parse(lat.Replace(',', '.'));
```
Comment (PieterL1976) [inline dcs-dtc/Models/FA18/FA18Configuration.cs:132]: Same as AH64D

File: dcs-dtc/Models/AH64/AH64Configuration.cs (hunk 1 above)
Comment (PieterL1976) [reply dcs-dtc/Models/AH64/AH64Configuration.cs:102]: Better coding is probably possible, just can't figure it out atm

**Review by PieterL1976 — CHANGES_REQUESTED** [review]
Make XML conversion more resilient

File: dcs-dtc/Models/AH64/AH64Configuration.cs (hunk 1 above)
Comment (Dobbie93) [reply dcs-dtc/Models/AH64/AH64Configuration.cs:102]: Unfortunately this is significantly outside my abilities and knowledge - I'll happily admit I was surprised I could even figure out why it wouldn't import on my machine

Comment (PieterL1976) [reply dcs-dtc/Models/AH64/AH64Configuration.cs:102]: OK, i'll pull it in with your other adjustment & make the alteration.

**Pull request merged**